forte/data/selector.py

```diff
@@ -85,14 +110,33 @@ def select(self, m_pack: MultiPack) -> Iterator[DataPack]:
                 f"Pack name {self.select_name}" f" not in the MultiPack"
             )
 
+    @classmethod
+    def default_configs(cls):
+        config = super().default_configs()
+        config.update(
+            {
+                "select_name": None
+            }
+        )
+        return config
+
 
 class RegexNameMatchSelector(SinglePackSelector):
     r"""Select a :class:`DataPack` from a :class:`MultiPack` using a regex."""
```

Comment:
Maybe add some docs here like in `NameMatchSelector` to show the previous and new usage.

Reply:
sounds good